base/setup/welcome/CMakeLists.txt

```diff
@@ -4,4 +4,24 @@ add_rc_deps(welcome.rc ${welcome_rc_deps})
 add_executable(welcome welcome.c welcome.rc)
 set_module_type(welcome win32gui UNICODE)
 add_importlibs(welcome gdi32 user32 shell32 msvcrt kernel32 ntdll)
+
+# Reduce the required subsystem to WinNT 4.0 for i386 builds only.
+if(ARCH STREQUAL "i386")
+    if(MSVC)
+        # add_target_link_flags(welcome "/SUBSYSTEM:WINDOWS,4.00")
```

Comment:
What's wrong with this? Why editbin?

Reply:
Because the commented line specifies the change at the level of the linker (link.exe for msvc), and that latter one checks and errors out if subsystem number is <= 5.0 in very recent msvc versions. Editbin on the contrary can use any version number, for example 4.0.